cmd/kpod/load.go

```diff
@@ -101,9 +105,19 @@ func loadCmd(c *cli.Context) error {
 // loadImage loads the image from docker-archive or oci to containers-storage
 // using the pullImage function
 func loadImage(store storage.Store, opts loadOptions) error {
-	systemContext := common.GetSystemContext("")
-
 	src := dockerArchive + opts.input
+	loadOpts := libkpodimage.CopyOptions{
+		Quiet:               opts.quiet,
+		OciImage:            opts.image,
+		Store:               store,
+		SignaturePolicyPath: "",
+	}
 
-	return libkpodimage.PullImage(store, src, false, opts.quiet, systemContext)
+	if err := libkpodimage.PullImage(src, false, loadOpts); err != nil {
```

Comment:
Can we look at the error coming back and tell if it's something fatal before we try loading as an OCI archive? There are probably errors where it doesn't make sense in proceeding, and if it's easy to check for these it might be fore it.

Reply:
there are a bunch of errors that might happen, it will require quite a lot of error constants I believe. I am not sure if its worth doing that

Reply:
OK, if it's too much effort and complexity it's not worth including.